lib/Runtime/Library/JavascriptError.cpp

```diff
@@ -47,12 +47,13 @@ namespace Js
         SetEnumerable(propertyId, false);
     }
 
-    Var JavascriptError::NewInstance(RecyclableObject* function, JavascriptError* pError, CallInfo callInfo, Var newTarget, Var message)
+    Var JavascriptError::NewInstance(RecyclableObject* function, JavascriptError* pError, CallInfo callInfo, Var newTarget, Var message, Var options)
     {
         ScriptContext* scriptContext = function->GetScriptContext();
 
         bool isCtorSuperCall = (callInfo.Flags & CallFlags_New) && newTarget != nullptr && !JavascriptOperators::IsUndefined(newTarget);
         JavascriptString* messageString = nullptr;
+        Var cause = nullptr;
```

Comment:
Could this be defined inside the `if` statement where it's used? it doesn't look like it's used outside of it.